vc-generator/issuer.js

```diff
@@ -41,7 +41,7 @@ export async function issueCloned({
   }
   return deriveCloned({
     verifiableCredential,
-    documentLoader: loader,
+    loader,
```

Comment:
It might make more sense to call the param `documentLoader` in these APIs.  That's more invasive and would break callers, but it would make the param name align through the stack.

Reply:
Agreed however this is from the jsigs library so would have to be tackled separately.